engine/src/main/java/org/terasology/rendering/world/WorldRendererImpl.java

```diff
@@ -438,18 +434,14 @@ private void renderObjectsOpaque() {
 
     private void renderChunksOpaque() {
         PerformanceMonitor.startActivity("Render Chunks (Opaque)");
-        while (renderQueues.chunksOpaque.size() > 0) {
-            renderChunk(renderQueues.chunksOpaque.poll(), ChunkMesh.RenderPhase.OPAQUE, playerCamera, ChunkRenderMode.DEFAULT);
-        }
+        renderChunk(renderQueues.chunksOpaque, ChunkMesh.RenderPhase.OPAQUE, playerCamera, ChunkRenderMode.DEFAULT);
```

Comment:
I like this simplification: good idea. However, the semantic of the method name is no longer correct. The method now deals with chunkS and should be renamed to renderChunks.